amazon_product_widget.install

```diff
@@ -40,6 +40,12 @@ function amazon_product_widget_schema() {
         'description' => 'Stores overrides for amazon product widget',
         'not null' => FALSE,
       ],
+      'available' => [
+        'description' => 'Represents product availability',
+        'type' => 'int',
+        'size' => 'tiny',
+        'default' => 0,
```

Comment:
if we do default 0, which makes sense, id suggest setting it to 1 for all existing entries in an update hook, and let the cron job slowly work out the unavailable ones.